tools/install/MergeModule.wxs

```diff
@@ -15,7 +16,13 @@
           <Directory Id="INSTALLDIR" Name="DNS-SD" FileSource="$(var.InstallSourceDir)">
             <Component Id="mDNSResponder" Guid="D555EC83-B046-4A39-8A4E-3FCF312820C6"
                 SharedDllRefCount="yes">
-              <File Id="mDNSResponderExe" Name="mDNSResponder.exe" DiskId="1" KeyPath="yes" />
+              <File Id="mDNSResponderExe" Name="mDNSResponder.exe" DiskId="1" KeyPath="yes">
+                <!-- Create Windows Firewall exceptions for the mDNSResponder executable. -->
+                <fire:FirewallException Id="EtcDnsSdTCP" Name="ETC DNS-SD Service (TCP-in)"
```

Comment:
Are these really only adding "in" rules for the ports?  I would think we need out rules as well.  Unless we've already tested this on a locked-down OS.

Reply:
Reading the WiX docs for the firewall element in use here, it seems to imply that it's creating inbound rules only:
"Path to a target program to be granted access to all incoming ports and protocols."
https://wixtoolset.org/documentation/manual/v3/xsd/firewall/firewallexception.html

Generally speaking, it has been my experience that by default the Windows Firewall allows outbound traffic and blocks inbound traffic. So in the default case, I believe that this will work as we expect. I feel like if someone is running a more restrictive firewall configuration than the default that they should account for what to let through that scenario rather than our telling them what they should allow through their configuration.

Reply:
@nwagner Making sure I got this right, should we try just keeping it as-is, creating just inbound rules for now, and see what testing comes up with?

Reply:
@samkearney Yes, leave it as is and see what test comes up with.